installer/win/util/brave_vpn_helper_utils.h

```diff
@@ -0,0 +1,34 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_INSTALLER_WIN_UTIL_BRAVE_VPN_HELPER_UTILS_H_
+#define BRAVE_INSTALLER_WIN_UTIL_BRAVE_VPN_HELPER_UTILS_H_
+
+#include <windows.h>
+#include <string>
+
+#include "base/files/file_path.h"
+#include "base/version.h"
+
+namespace brave_vpn {
+
+bool ConfigureServiceAutoRestart(const std::wstring& service_name,
```

Comment:
why is this public? It doesn't appear to be called outside of brave_vpn_helper_utils